### PR TITLE
Added update-api scenarios

### PR DESCRIPTION
**Pull request by rheafernandes**

### CODE REVIEW

File: kp_service_test/src/test/resources/templates/content/v3/update/testUpdateEcmlBodyForPublishedContent/response.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "resmsgid": "@ignore@",
+    "msgid": null,
+    "err": null,
+    "status": "successful",
+    "errmsg": null
+  },
+  "responseCode": "OK",
+  "result": {
+    "node_id": "${contentIdVal}",
+    "versionKey": "@ignore@"
```
Comment (krgauraw) [inline kp_service_test/src/test/resources/templates/content/v3/update/testUpdateEcmlBodyForPublishedContent/response.json]: please use @isNumber()@

File: kp_service_test/src/test/resources/templates/content/v3/update/testUpdateContentNewFramework/validate.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      "idealScreenSize": "normal",
+      "createdOn": "@ignore@",
+      "appId": "@ignore@",
+      "contentDisposition": "inline",
+      "contentEncoding": "identity",
+      "lastUpdatedOn": "@ignore@",
+      "contentType": "Resource",
+      "dialcodeRequired": "No",
+      "identifier": "@ignore@",
+      "lastStatusChangedOn": "@ignore@",
```
Comment (krgauraw) [inline kp_service_test/src/test/resources/templates/content/v3/update/testUpdateContentNewFramework/validate.json]: please use @matchesDatePattern('yyyy-MM-dd')@

File: kp_service_test/src/test/resources/templates/content/v3/update/testUpdateEcmlBodyForPublishedContent/response.json (hunk 1 above)
Comment (rheafernandes) [reply kp_service_test/src/test/resources/templates/content/v3/update/testUpdateEcmlBodyForPublishedContent/response.json]: Fixed this

File: kp_service_test/src/test/resources/templates/content/v3/update/testUpdateContentNewFramework/validate.json (hunk 2 above)
Comment (rheafernandes) [reply kp_service_test/src/test/resources/templates/content/v3/update/testUpdateContentNewFramework/validate.json]: Fixed this

**Pull request merged**